modules/social_features/social_group/modules/social_group_request/src/SocialGroupRequestConfigOverride.php

```diff
@@ -87,19 +87,6 @@ public function loadOverrides($names) {
     $outsider_role_configs = [];
     foreach ($social_group_types as $social_group_type) {
       $default_form_display_configs[] = "core.entity_form_display.group.{$social_group_type}.default";
-      $outsider_role_configs[] = "group.role.{$social_group_type}-outsider";
```

Comment:
Can you explain where this logic was moved?

Reply:
It was removed to `modules/social_features/social_group/modules/social_group_flexible_group/config/install/group.role.flexible_group-verified.yml` file as default permission, and also it was added in `social_group_flexible_group_update_130010` update